src/scarlett_mixer.c

```diff
@@ -631,6 +628,10 @@ static int open_mixer (RobTkApp* ui, const char* card, int opts)
 		assert (i <= cnt);
 	}
 
+  if ((opts & OPT_DETECT) && d.matrix_mix_column_major) {
+    d.matrix_mix_stride = d.smi;
+  }
+
```

Comment:
`matrix_in_stride` is not set anymore (removed above). Also please use tabs for indent (and space to align).

Reply:
It will be set in an earlier branch. In my case, it's set correctly to 1 (instead of 10 being set here)